client/src/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,61 @@
+@import "./global.scss";
+
+body {
+  background-color: $yellow;
+  text-align: center;
+  height: $page-height;
+}
+.avatar {
+  width: 30px;
+}
+.dashboard-h1 {
+  padding-top: 0;
+  margin-top: 0;
+}
+.dash-container {
+  background-color: $yellow;
+  text-align: center;
+  float: none;
+  margin: 0 auto;
+  color: $white;
+  font-size: $header;
+  font-family: $font-secondary;
+}
+.container {
+  padding-top: 15vh;
+}
+.col-sm {
+  height: 15vh;
+  margin: 5vh;
+  background-color: $black;
+  text-align: center;
+  -webkit-box-shadow: -1px 0px 22px -8px rgba(0, 0, 0, 1);
+  -moz-box-shadow: -1px 0px 22px -8px rgba(0, 0, 0, 1);
+  box-shadow: -1px 0px 22px -8px rgba(0, 0, 0, 1);
+}
+.btn-dash {
+  color: $white;
+  background-color: $black;
+  width: 30vh;
+  font-size: $btn-size;
+  margin: 5vh;
+  font-family: $font-main;
+}
+.btn-logout {
+  color: $blue;
+  border: none;
+  background-color: $yellow;
+  width: 30vh;
+  font-size: 26pt;
+  font-family: $font-main;
+  position: absolute;
+  left: 160vh;
+  top: 92vh;
+}
+.clock {
```

Comment:
It would make sense to keep component specific styles in their own component instead of globally styling it. [Here are 5 cool techniques](https://blog.bitsrc.io/5-ways-to-style-react-components-in-2019-30f1ccc2b5b) to style your react component